outscale/framework_config.go

```diff
@@ -98,15 +102,15 @@ func isProfileSet(data *ProviderModel) (bool, error) {
 		}
 		jsonFile, err := os.ReadFile(configFilePath)
 		if err != nil {
-			return isProfSet, fmt.Errorf("%v \n Connot found configue file: %v", err, configFilePath)
+			return isProfSet, fmt.Errorf("%w \n Cannot found configue file: %v", err, configFilePath)
```

Comment:
Cannot read config file (the file may exist and not be readable)

It is usual to start with your message and end with the error message.